src/assets/toolkit/scripts/lib/component/slideshow1.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+export class Slideshow {
+  constructor (element, {
+    slideHolder = element.querySelector('.js-Slideshow-slides'),
+    slides = element.querySelectorAll('.js-Slideshow-slide'),
+    nextTrigger = element.querySelector('.js-Slideshow-next'),
+    prevTrigger = element.querySelector('.js-Slideshow-prev'),
+    currentCount = element.querySelector('.js-Slideshow-current'),
+    totalCount = element.querySelector('.js-Slideshow-total'),
+    classForward = 'slide-forward',
+    classForwardReset = 'slide-forwardReset',
+    classBack = 'slide-back'
+  } = {}) {
+
+    this.slides = Array.from(slides);
```

Comment:
You might consider using [Umbrella](http://umbrellajs.com/) (already a dependency) which would render this sorta stuff unnecessary.